Index: encryptroot-asahi/src/encryptroot.asahi
===================================================================
--- encryptroot-asahi.orig/src/encryptroot.asahi
+++ encryptroot-asahi/src/encryptroot.asahi
@@ -96,18 +96,6 @@ check_arguments() {
     [ -n "$BOOTDISK" ] || exit_error "missing mandatory argument BOOTDISK"
 }
 
-check_dependencies() {
-    if [ -z "$EXT4_FS" ]; then
-        RESIZE_PROGS="btrfs"
-    else
-        RESIZE_PROGS="e2fsck resize2fs"
-    fi
-    # shellcheck disable=2086
-    for c in arch-chroot cryptsetup lsblk $RESIZE_PROGS uuidgen; do
-        command -v $c >/dev/null 2>&1 || exit_error "command '$c' not installed or not on path"
-    done
-}
-
 explain_and_warn() {
     echo "*** WARNING ***"
     echo
@@ -421,7 +409,6 @@ chroot_update_initramfs() {
 if [ -z "$ENCRYPTROOT_ASAHI_CHROOT_43F12E50" ]; then
     check_root
     check_arguments
-    check_dependencies
     explain_and_warn
     check_disks
 
